src/achievement.cpp

```diff
@@ -335,6 +354,10 @@ class requirement_watcher : stat_watcher
             stats.add_watcher( req.statistic, this );
         }
 
+        const cata_variant current_value() const {
```

Comment:
Clang tidy caught this:
```suggestion
        cata_variant current_value() const {
```